src/main/java/Collections/LinkedList_Example.java

```diff
@@ -51,6 +51,7 @@ public static void main(){
     void TestRayIsLastItemIntheList(){
         Assertions.assertTrue(students.getLast() == "Ray");
         Assertions.assertTrue(students.contains("Ray"));
+        Assertions.assertTrue(students.getLast().equals("Ray"));
```

Comment:
@rayme11 please take a look at the first and last line of these assertions - I think we are repeating the same intent: asserting that the string object is "Ray".